test/run.test.ts

```diff
@@ -18,13 +18,20 @@ const travelPageFeatureGetter = {
   },
 };
 
+const lookBackPageFeatureGetter = {
+  name: 'keywords',
+  func: (): Promise<string[]> => {
+    return Promise.resolve(['lb']);
```

Comment:
I think this is a little confusing to someone else, "why do we need 'lb' keyword here"

Might be good idea to match all page views when keywords are empty?

WDYT @Risto-Stevcev @k11kirky

Reply:
ok, thats fine to me